src/bin/scribble.ts

```diff
@@ -151,8 +150,17 @@ function ppWarning(warn: Warning): string[] {
     return [`${ppLoc(start)} Warning: ${warn.msg}`, getSrcLine(warn.location)];
 }
 
+function assertIsFile(fileName: string): void {
+    assert(fileName !== "", "Path is empty");
+    assert(fse.existsSync(fileName), "Path not found: " + fileName);
+
+    const stats = fse.statSync(fileName);
```

Comment:
This is not a blocker - just a small optimization. We can remove the call to `fse.existsSync` here by doing:
```
assert(fileName !== "", "Path is empty");
try {
        const stats = fse.statSync(fileName);
        assert(stats.isFile(), "Target is not a file: " + fileName);
} catch (e) {
         assert(false, e.code === 'ENOENT' ? ("Path not found: " + fileName) :  e.message);
}
```

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/164/commits/42b350e912bd4eef2f59dc60d3bdfcb6f27e67d3